install.sh

```diff
@@ -54,9 +54,6 @@
 #
 # __________ Process command-line arguments and environment variables _____________
 
-# Make sure we really exit with correct status when piping function output
-set -o pipefail
-
```

Comment:
@rouson is this being intentionally removed? This is needed to ensure that the correct exit status is returned when the script fails.

I'm thinking maybe you need to merge master into your branch (or git rebase master) since this was changed in  a recent commit.